__tests__/app.test.js

```diff
@@ -305,4 +305,78 @@ describe("PATCH /api/reviews/:review_id", () => {
     })
 })
 
+describe("GET /api/reviews", () => {
+    it("200: Endpoint can accept additional queries. If 'category' is passed then endpoint should only return reviews with the corresponding category", () => {
+        return request(app)
+        .get("/api/reviews?category=dexterity")
+        .expect(200)
+        .then((response) => {
+            const returnedReview = response.body.reviews
+            expect(returnedReview.length).toBe(1)
+            returnedReview.forEach((review) => {
+                expect(review.category).toBe("dexterity")
+                expect(review.category).not.toBe("euro game")
+                expect(review.category).not.toBe("social deduction")
+            })
+        })
```

Comment:
Please add another test (200) where the category is valid, but there are no reviews (e.g. ?category=children's games) so it responds with an empty array.